packages/neuron-wallet/src/Channel.ts

```diff
@@ -130,13 +130,17 @@ const listenToChannel = () => {
    * @name GetTransactions
    * @description get transactions
    */
-  ipcMain.on(Channel.GetTransactions, (e: Electron.Event) => {
-    console.info('get transactions')
-    setTimeout(() => {
-      e.sender.send(Channel.GetTransactions, {
-        status: 1,
-        result: ['transaction'],
-      })
+  ipcMain.on(Channel.GetTransactions, (e: Electron.Event, { page, pageSize }: { page: number; pageSize: number }) => {
+    console.info(`get transactions in page: ${page}, pageSize: ${pageSize}`)
+    e.sender.send(Channel.GetTransactions, {
+      status: 1,
+      result: {
+        count: transactions.length * 2,
```

Comment:
What does `transactions.length * 2` mean?

Reply:
Used to mock total count of transactions, moved to `mock` in this [PR](https://github.com/nervosnetwork/neuron/pull/92/commits/c7259636b1cbd30d79f6249025bd974f01686160#diff-b472b3d95a07b1552f8c755a48785478L4)